ci-operator/config/openshift/assisted-test-infra/openshift-assisted-test-infra-release-ocm-2.4.yaml

```
@@ -28,6 +28,10 @@ images:
   from: src
   to: assisted-test-infra
 - dockerfile_path: Dockerfile.assisted-test-infra
+  inputs:
+    assisted-service:
+      as:
+      - quay.io/edge-infrastructure/assisted-service:ocm-2.4
```

not sure I understand that one
looking on test-infra dockerfile: https://github.com/openshift/assisted-test-infra/blob/release-ocm-2.4/Dockerfile.assisted-test-infra#L1

meaning this substitution will have no effect, and you probably meant:
```suggestion
      - quay.io/ocpmetal/assisted-service:latest
```